src/bci_build/package/__init__.py

```diff
@@ -1435,6 +1441,97 @@ def prepare_template(self) -> None:
         pass
 
 
+class ContainerFamily:
+    """ContainerFamily is grouping multiple containers build flavors.
+
+    This provides package-central functions like generating _service and
+    _multibuild files, checking version_uid
+    """
+
+    def __init__(
+        self,
+        containers: Sequence[
+            BaseContainerImage | DevelopmentContainer | ApplicationStackContainer
```

Comment:
`BaseContainerImage` is the base class of the latter two:
```suggestion
            BaseContainerImage
```

Reply:
fixed.